cluster-api-gcp/deploy/deploy_helper.go

```diff
@@ -234,18 +235,23 @@ func (d *deployer) getMasterIP(master *clusterv1.Machine) (string, error) {
 }
 
 func (d *deployer) copyKubeConfig(master *clusterv1.Machine) error {
-	for i := 0; i <= RetryAttempts; i++ {
-		var config string
-		var err error
-		if config, err = d.machineDeployer.GetKubeConfig(master); err != nil || config == "" {
-			glog.Infof("Waiting for Kubernetes to come up...")
-			time.Sleep(time.Duration(SleepSecondsPerAttempt) * time.Second)
-			continue
+	var config string
+	var err error
+	eb := util.NewExponentialBackoff(time.Duration(SleepSecondsPerAttempt) * time.Second * RetryAttempts, 10 * time.Second)
```

Comment:
The RetryAttempts const is no longer accurate to the behavior given that it is now using exponential backoff. Perhaps a different const should be used for logic using exponential backoff while you migrate RetryAttempts stuff over to this new behavior?